omero/developers/Java.txt

```diff
@@ -694,6 +732,24 @@ to an image.
       } else if (shape instanceof PointData) {
         PointData pointData = (PointData) shape;
         // Handle point
+      } else if (shape instanceof MaskData) {
+        MaskData maskData1 = (MaskData) shape;
+        //Handle mask
+      }
+
+      //Check if the shape has transform
+      //http://blog.openmicroscopy.org/data-model/future-plans/2016/06/20/shape-transforms/
+      AffineTransformI transform = shape.getTransform();
+      if (transform != null){
+
+        double xScaling = transform.getA00.getValue();
+        double xShearing = transform.getA01.getValue();
+        double xTranslation = transform.getA02.getValue();
+            
+        double yScaling = transform.getA11.getValue();
+        double yShearing = transform.getA10.getValue();
+        double yTranslation = transform.getA12.getValue();
+        // Handle transforms
```

Comment:
Could we expand the comment a bit then to clarify that? "Here your own code can ... " or something. At the moment it looks just like others (like for the statements above) that are on sections which are complete as-is.